platform_disk_api/kube_client.py

```diff
@@ -487,17 +503,21 @@ async def watch_pods(
                 pass
 
     async def get_pvc_volumes_metrics(self) -> AsyncIterator[PVCVolumeMetrics]:
+        assert self._client, "client is not initialized"
         # Get list of all nodes
         nodes_url = f"{self._api_v1_url}/nodes"
-        payload = await self._request(method="GET", url=nodes_url)
-        self._raise_for_status(payload)
+        async with self._client.request(method="GET", url=nodes_url) as response:
```

Comment:
curious why we avoid using `_request` here?

Reply:
calling the proxy api below using `_client` was fine I guess.

Reply:
Forgot the reason, and thought I had a brain bug.
Previously changed it since _raise_for_status relies on response payload structure (which should include `kind`), but the metrics endpoint does not include it.

I will revert [6be1dcf](https://github.com/neuro-inc/platform-disk-api/pull/426/commits/6be1dcf2ad5ad86fac584801f5e372d8df9bc4ca) and call client directly for fetching metrics.